.well-known/ecips/specs.json

```diff
@@ -0,0 +1,21 @@
+---
+layout: null
+---
+
+[
+    {% for page in site.specs %}
+    {
+        "uri": "ecips.ethereumclassic.org/{{ page.slug }}",
+        "documentUrl": "https://ecips.ethereumclassic.org{{ page.id }}",
```

Comment:
is the `/` missing here before the page.id or is it added from the `_config.yml`?

Reply:
No. The above is correct. `page.id` contains a leading `/`.